exporter/awsprometheusremotewriteexporter/factory.go

```diff
@@ -56,18 +54,8 @@ func (af *awsFactory) CreateDefaultConfig() configmodels.Exporter {
 
 	cfg.TypeVal = typeStr
 	cfg.NameVal = typeStr
-
 	cfg.HTTPClientSettings.CustomRoundTripper = func(next http.RoundTripper) (http.RoundTripper, error) {
-		if !isAuthConfigValid(cfg.AuthConfig) {
-			return nil, errors.New("invalid authentication configuration")
-		}
```

Comment:
The original intention of this was to throw an error in the case that a user missed a field in auth config, like this:
```
awsprometheusremotewrite:
  ...
  aws_auth:
    region: us-west-2
```

Now instead this configuration would simply result in the request not being signed, which will produce a 403 forbidden error I believe. If we believe that error message is good enough for customers to recognize where the issue is then I am happy with this change